host-services/builtins/messaging.go

```diff
@@ -55,21 +55,27 @@ func (m *MessagingService) Initialize(config json.RawMessage) error {
 }
 
 func (m *MessagingService) HandleRequest(
-	nc *nats.Conn,
+	conns []*nats.Conn,
 	namespace string,
 	workloadId string,
 	method string,
 	workloadName string,
 	metadata map[string]string,
 	request []byte) (hostservices.ServiceResult, error) {
 
+	conn := conns[0]
+	// if there's a separate "trigger" connection, we use that to ensure consistency
```

Comment:
This is a little confusing, that is, that the trigger connection is always at `[1]`.

Reply:
yeah I was going to add numeric constants for those indexes but it seemed like overkill for just one.
I had a previous incarnation of this that used multiple discrete connection parameters but I figured that would get ugly if we needed yet another connection.